indico/modules/events/editing/controllers/base.py

```diff
@@ -25,6 +25,10 @@
 class TokenAccessMixin(object):
     SERVICE_ALLOWED = False
 
+    def __init__(self):
+        super(TokenAccessMixin, self).__init__()
+        self.is_service_call = False
```

Comment:
No need for this, you can just add `is_service_call = False` above below `SERVICE_ALLOWED = False`. It's not mutable so nothing wrong with setting the default on the class.

Reply:
Of course, static here doesn't really apply. Thanks!